pkg/util/issutil/config.go

```diff
@@ -160,13 +165,14 @@ func DefaultParams(initialMembership map[t.NodeID]t.NodeAddress) *ModuleParams {
 	// Define auxiliary variables for segment length and maximal propose delay.
 	// PBFT view change timeouts can then be computed relative to those.
 	return (&ModuleParams{
-		InitialMembership:  initialMembership,
-		ConfigOffset:       2,
-		SegmentLength:      4,
-		NumBuckets:         len(initialMembership),
-		RequestNAckTimeout: 16,
-		MsgBufCapacity:     32 * 1024 * 1024, // 32 MiB
-		RetainedEpochs:     1,
+		InitialMembership:     initialMembership,
+		ConfigOffset:          2,
+		SegmentLength:         4,
+		NumBuckets:            len(initialMembership),
+		RequestNAckTimeout:    16,
+		MsgBufCapacity:        32 * 1024 * 1024, // 32 MiB
+		RetainedEpochs:        1,
+		LeaderSelectionPolicy: Simple,
```

Comment:
Why use Simple not Blacklist?

Reply:
I made the `Simple` policy the default one to not break the interface, the `Simple` policy has implicitly been the default one before, as there was no other one. We can use `Blacklist` as the default in the future, but for now I'd prefer to have it "opt-in" rather than "opt-out".